docs: tw798 (#3270)
OffchainLabs/arbitrum-docs: docs/for-devs/third-party-docs/Envio/envio.mdx

## Envio HyperIndex

[Envio](https://envio.dev/) HyperIndex is a feature-rich indexing solution that provides developers with an efficient way to index and aggregate real-time or historical blockchain data for any EVM. The indexed data is easily accessible through custom GraphQL queries, providing developers with the flexibility and power to retrieve specific information.

> Envio offers native support for Arbitrum One (testnet & mainnet), Aribitrum Nova (testnet & mainnet), and Arbitrum chains, and has been designed to support high-throughput blockchain applications that rely on real-time data for their business requirements.

Reviewer: ```suggestion
Envio offers native support for Arbitrum One (testnet and mainnet), Arbitrum Nova (testnet and mainnet), and Arbitrum chains, and has been designed to support high-throughput blockchain applications that rely on real-time data for their business requirements.
```

- **Multi-chain Support**: Aggregate data across multiple networks into a single database. Query all your data with a unified GraphQL API.

> - **Asynchronous Mode**: Fetch data from offchain storage such as IPFS, or contract state (e.g., smart contract view functions).

Reviewer: ```suggestion
- **Asynchronous Mode**: Fetch data from offchain storage such as IPFS, or contract state (for example, smart contract view functions).
```

> - **Quickstart Templates**: Use pre-defined indexing logic for popular OpenZeppelin contracts (e.g., **ERC-20**).

Reviewer: ```suggestion
- **Quickstart Templates**: Use pre-defined indexing logic for popular OpenZeppelin contracts (for example, `ERC-20`).
```
